src/main/java/com/google/opensesame/projects/ProjectData.java

```diff
@@ -158,4 +159,17 @@ public String getRepositoryId() {
 
     return Collections.unmodifiableList(mentors);
   }
+
+  /**
+   * Gets the URL of the GitHub page for the repository.
+   *
+   * @return Returns the URL of the GitHub page.
+   */
+  public String getGitHubHtmlUrl() {
```

Comment:
Just wondering why we have Html in there? Doesn't seem to have to do with Html to me, but I might be misunderstanding something.

Reply:
You make a good point. I thought it'd be best to match the naming used in the Java GitHub API, but on second thought I think I'll change it, as it doesn't seem necessary. Thanks!